### PR TITLE
Css canvas medical- David Gonzalez

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
https://user-images.githubusercontent.com/24907191/168505789-9096937d-2bd1-42ba-82ab-1d0ba4907e97.mov

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        <button class="contact-us-btn">Product</button>
+        <button class="contact-us-btn">Implemenation</button>
+        <button class="contact-us-btn">About Us</button>
+        <button class="contact-us-btn">Blog</button>
+        <button class="contact-us">Contact Us</button>
+      </section>
+    </div>
+
+
+
```
Comment (Zhiwutian) [inline css-canvas-medical/index.html:26]: make sure to remove any whitespace in the code like this.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking solid overall! A few things to look at:
1. Make sure to remove excess whitespace from the code.
2. Make sure the widths for your column classes match the names, for example 2/3 should indicate 2/3 of a rows width. 
3. some of the text spacing needs to be adjusted. for example the address in the footer. Make sure to check the text spacing of the example.

Keep up the good work!

**Comment by David-Gonzalez11** [comment]
https://user-images.githubusercontent.com/24907191/168701303-bd4ece74-d701-445d-ac27-03642b5eb0b0.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**